{
  "uid" : "6500dcb37a34e549",
  "name" : "test_login_usernameNull_error",
  "fullName" : "TestCases.test_logincase.test_login.Test_Login#test_login_usernameNull_error",
  "historyId" : "e3de2cc26aceee9068f98fa7cfd0be9f",
  "time" : {
    "start" : 1610619072742,
    "stop" : 1610619072742,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "AttributeError: 'NoneType' object has no attribute 'maximize_window'",
  "statusTrace" : "@pytest.fixture(scope='module')\n    def Driver():\n        driver = Webdriver()\n>       driver.maximize_window()\nE       AttributeError: 'NoneType' object has no attribute 'maximize_window'\n\nTestCases\\conftest.py:18: AttributeError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "Driver",
    "time" : {
      "start" : 1610619072743,
      "stop" : 1610619079547,
      "duration" : 6804
    },
    "status" : "broken",
    "statusMessage" : "AttributeError: 'NoneType' object has no attribute 'maximize_window'\n",
    "statusTrace" : "  File \"D:\\Anaconda3\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\Anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\Anaconda3\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"D:\\Git\\PycharmProjects\\PycharmProjects\\Reptile\\Web_Framework-master\\TestCases\\conftest.py\", line 18, in Driver\n    driver.maximize_window()\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "stepsCount" : 0,
    "hasContent" : true
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "AttributeError: 'NoneType' object has no attribute 'maximize_window'",
    "statusTrace" : "@pytest.fixture(scope='module')\n    def Driver():\n        driver = Webdriver()\n>       driver.maximize_window()\nE       AttributeError: 'NoneType' object has no attribute 'maximize_window'\n\nTestCases\\conftest.py:18: AttributeError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "956924c9228f959e",
      "name" : "log",
      "source" : "956924c9228f959e.txt",
      "type" : "text/plain",
      "size" : 990
    } ],
    "parameters" : [ ],
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "stepsCount" : 0,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "story",
    "value" : "测试账号为空"
  }, {
    "name" : "parentSuite",
    "value" : "TestCases.test_logincase"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "subSuite",
    "value" : "Test_Login"
  }, {
    "name" : "host",
    "value" : "USER-20190821YP"
  }, {
    "name" : "thread",
    "value" : "10252-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "TestCases.test_logincase.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "6500dcb37a34e549.json",
  "parameterValues" : [ ]
}